leader-follower/src/main/java/com/iluwatar/leaderfollower/App.java

```diff
@@ -0,0 +1,59 @@
+package com.iluwatar.leaderfollower;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * 
+ * Leader Follower is a concurrency pattern where multiple threads can efficiently demultiplex
+ * events and dispatch to event handlers. One may think of it as a taxi station at night, where all
+ * the drivers are sleeping except for one, the leader. The ThreadPool is a station managing many
+ * threads - cabs.
+ * 
+ * The leader is waiting for an IO event on the HandleSet, so as a driver waits for a client.
+ * 
+ * When a client arrives (in the form of a Handle identifying the IO event), the leader driver wakes
+ * up another driver to be the next leader and serves the request from his passenger.
+ * 
+ * While he is taking the client to the given address (calling ConcreteEventHandler and handing over
+ * Handle to it) the next leader can concurrently serve another passenger.
+ * 
+ * When a driver finishes he take his taxi back to the station and falls asleep if the station is
+ * not empty. Otherwise he become the leader
```

Comment:
Fix typos from the text. When referencing other classes in Javadoc use {@link ClassName} notation.